pkg/render/intrusion_detection_test.go

```diff
@@ -420,9 +420,16 @@ var _ = Describe("Intrusion Detection rendering tests", func() {
 			},
 			{
 				APIGroups: []string{"authorization.k8s.io"},
-				Resources: []string{"subjectaccessreviews"},
+				Resources: []string{"tokenreviews"},
```

Comment:
previously checking the wrong resource

Reply:
Could you perhaps change the NotTo() to To(), because the NotTo is setting us up for future issues.

Reply:
Added the cronjobs/jobs rbac check to an existing test that checks it contains with To() in a management/standalone cluster setup https://github.com/tigera/operator/pull/2065/commits/45ba5b15d87c7ab28faf5c6449a2ade371699ba1 -